### PR TITLE
Feat: Added argument splitter for grouped short options

### PR DESCRIPTION
**Pull request by hwittenborn**
# Purpose

Previously, arguments for short options passed to Pacstall were not allowed to be grouped together (i.e. you had to do `-P -I` instead of `-PI`, as `-PI` wouldn't be recognized as an option).

# Approach

Now, any options that start with a single hyphen are automatically split on every occurrence of a capital letter.

Likewise:

`-PI` would become `-P -I`

`-PIl` would become `-P -Il`

`-IlP` would become `-Il -P`

### CODE REVIEW

**Comment by hwittenborn** [comment]
Looks like I forgot to remove the echo and exit commands :P

I'll get that fixed later today

File: pacstall [hunk 1]
```diff
@@ -205,6 +205,26 @@ fi
...
+		continue
+	fi
+
+	# Add all arguments to the list of arguments.
+	# We remove the '-' prefix as we'll add it later.
+	for j in $(echo "${i}" | sed 's|^-||' | sed 's|[[:upper:]]| &|g'); do
+		argument_list+=("-${j}")
+	done
+done
+
```
Comment (D-Brox) [inline pacstall:224]: If the new arguments list has a `-P` you should move it to the start so that the prompts are disabled properly

**Review by D-Brox — CHANGES_REQUESTED** [review]

**Comment by D-Brox** [comment]
Also, can you fetch upstream so that the update script is fixed on your branch? That way we can test it more easily

File: pacstall (hunk 1 above)
Comment (hwittenborn) [reply pacstall:233]: Done

**Comment by hwittenborn** [comment]
> Also, can you fetch upstream so that the update script is fixed on your branch? That way we can test it more easily

All done

**Review by D-Brox — APPROVED** [review]

**Pull request merged**